pkg/graph/schema.graphql

```diff
@@ -1060,6 +1060,21 @@ type SystemIntake {
   trbFollowUpRecommendation: SystemIntakeTRBFollowUp
   contractName: String
   relationType: SystemIntakeRelationType # TODO: NOT IMPLEMENTED
+
+  """
+  Linked contract numbers
+  """
+  contractNumbers: [SystemIntakeContractNumber!]!
+}
+
+type SystemIntakeContractNumber {
+  id: UUID!
+  systemIntakeID: UUID!
+  contractNumber: String!
+  createdBy: UUID!
+  createdAt: Time!
+  modifiedBy: UUID!
+  modifiedAt: Time!
```

Comment:
Might just be worth making sure there's some alignment between BaseStruct/SQL Table/This type in GQL with regards to the null-ability of these fields.

Reply:
updated! Great call - thank you Clay!